projects/dsp-ui/src/lib/search/search-panel/search-panel.component.html

```diff
@@ -44,9 +44,8 @@ <h4 *ngIf="!showAdvanced">Expert search</h4>
             </span>
         </div>
         <div class="dsp-menu-content">
-            <!-- TODO: reactivate this tags when the components are ready -->
-            <!-- <dsp-advanced-search *ngIf="showAdvanced" (search)="search.emit($event); closeMenu()"></dsp-advanced-search> -->
-            <!-- <dsp-expert-search *ngIf="!showAdvanced" (search)="search.emit($event); closeMenu()"></dsp-expert-search> -->
+            <dsp-advanced-search *ngIf="showAdvanced" (search)="search.emit($event); closeMenu()"></dsp-advanced-search>
```

Comment:
Could you put `search.emit($event); closeMenu()` in a method? So then you could call in both cases: advanced and expert search.

Reply:
fixed in 139009f